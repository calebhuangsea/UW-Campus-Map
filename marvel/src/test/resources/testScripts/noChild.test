# Find a path where child node doesn't exist

LoadGraph noChild noChild.csv

FindPath noChild n1 n2