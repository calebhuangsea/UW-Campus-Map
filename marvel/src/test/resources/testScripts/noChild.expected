# Find a path where child node doesn't exist

loaded graph noChild

unknown: n2